src/wp-includes/default-constants.php

```diff
@@ -133,6 +133,15 @@ function wp_initial_constants() {
 	define( 'MONTH_IN_SECONDS',  30 * DAY_IN_SECONDS    );
 	define( 'YEAR_IN_SECONDS',  365 * DAY_IN_SECONDS    );
 	/**#@-*/
+
+	/**#@+
+	 * Constants for CP_STRICT.
+	 *
+	 * @since 1.2.0
+	 */
+	define( 'CP_STRICT_HOOK_ARGS_DECL',    0X00000001 );
+	define( 'CP_STRICT_HOOK_ARGS_RUNTIME', 0X00000002 );
+	/**#@-*/
```

Comment:
It doesn't make sense to do this as a bitmask, because you can't use these values in `wp-config.php` (they're not defined yet). This is also more difficult to explain.

Should be done as individual boolean flags instead. Usage variations are as follows:

```
define( 'CP_STRICT', true ); // everything enabled

define( 'CP_STRICT', true );
define( 'CP_STRICT_HOOK_ARGS_DECL', false ); // everything enabled except this
```